MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs

```diff
@@ -263,6 +265,23 @@ public string ErrorMessage
                 OnPropertyChanged("HasError");
             }
         }
+
+
+
+        public bool IsLoading
```

Comment:
Isn't this the same as IsBusy, which is already on there?

Reply:
Good point.